pyautosplit/process.py

```diff
@@ -41,3 +45,6 @@ def read_int(self, addr):
 
     def read_bool(self, addr):
         return self.dprocess.readBytes(addr, 1) == b'\01'
+
+    def read_pointer(self, addr):
+        return int.from_bytes(self.dprocess.readBytes(addr, 8), 'little')
```

Comment:
This is basically the same function as `read_int`, only the length is 8. I would suggest using the `read_int` function of your other PR. (I will get to that later)

Reply:
You're right, this is a relict from my programming order.